sentence-generator.js

```diff
@@ -47,16 +93,18 @@ const sentence = () =>{
         break;
     }
     
-    let FINAL_STRING = sentenceArray.join(' '); //convert sentence array into a string
+    const FINAL_STRING = sentenceArray.join(' '); //convert sentence array into a string
     
 
-    document.getElementById("result").value = FINAL_STRING; // send it to index.html
+    document.getElementById('result').innerHTML = FINAL_STRING; // send it to index.html
     
     
     
 }
 
+let button = document.getElementById('button');
 
```

Comment:
Love that you broke this up to make it more readable, but you can use a `const` here

Reply:
Also, this value can be pre-computed before your code runs, consider moving it to the top of the file